examples/forms.amp.html

```diff
@@ -535,5 +535,18 @@ <h4 id="header1">Using button[type=submit]</h4>
 </form>
 
 
+<h4 id="header1">Redirect After POST</h4>
+<form method="post"
+      action-xhr="/form/redirect-to/post"
```

Comment:
Can you make the request to e.g. `iframe.localhost` (which we mandate to exist) to actually simulate CORS?